src/core/killall.c

```diff
@@ -177,8 +178,11 @@ static int killall(int sig, Set *pids, bool send_sighup) {
                 }
 
                 if (kill(pid, sig) >= 0) {
-                        if (pids)
-                                set_put(pids, ULONG_TO_PTR(pid));
+                        if (pids) {
+                                r = set_put(pids, ULONG_TO_PTR(pid));
+                                if (r < 0)
+                                        log_oom();
```

Comment:
Well, the only reason log_oom() returns -ENOMEM is that you can write "return log_oom()"... The return value hence is mostly just a nicety, and nothing we consider something crucial that shouldn't be ignore. Hence in cases like this there's no probably should *not* cast to (void).

Or in other words, Casting a call to "(void)" is something we should do for calls where it's uncommon or often risky if the error code is ignored, and we want to indicate that we thought about the issue and decided to ignore the return code anyway.